### PR TITLE
Azure Manifest tests for service availability and pipeline execution

### PR DESCRIPTION
**Pull request by yilun-msft**
Created pipelines to monitor the service availability and pipeline execution for Azure Manifest on Kubeflow.org. The tests are two-folds. First to check service availability, i.e. KF pods are ready and healthy. Second to check a pipeline artifact could be created and executed. These sets of tests would be used to constantly monitor Azure Manifest and make sure Kubeflow could be installed on AKS and a simple set of pipeline could be executed.

### CODE REVIEW

File: .github/workflows/manifest_test.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      - name: Create Resource Group
+        run: |
+          # az login using service principal
+          az login --service-principal --username ${{ secrets.APP_ID }} --password ${{ secrets.PASSWORD }} --tenant ${{ secrets.TENANT_ID }}
+          az group create -n kf-test-rg -l westus2
+      - name: Install curl
+        run: sudo apt install curl
+      - name: Apply Resources
+        run: |
+          # Create AKS resources
```
Comment (eedorenko) [inline .github/workflows/manifest_test.yml:43]: That's too long for an inline script from readability perspective. Can you move it to a shell script and define all used constants (like URLs, Istio version) at the top of the script?

File: code/tests/manifest_tests.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            name='Testing pipeline',
+            description='Testing pipeline'
+            )
+    def test_pipeline():
+        my_step = op()
+        print(my_step.name)
+
+    kfp.compiler.Compiler().compile(test_pipeline, 'test-pipeline.zip')
+
+    client = kfp.Client()
```
Comment (eedorenko) [inline code/tests/manifest_tests.py:26]: Does it work fro you? Without any parameters?

**Review by eedorenko — CHANGES_REQUESTED** [review]

File: code/tests/manifest_tests.py (hunk 2 above)
Comment (yilun-msft) [reply code/tests/manifest_tests.py:26]: Yup. It passed the tests: https://github.com/yilun-msft/kubemlops/actions?query=workflow%3AManifest_Test

File: .github/workflows/manifest_test.yml (hunk 1 above)
Comment (yilun-msft) [reply .github/workflows/manifest_test.yml:43]: Sure. Will update

File: .github/workflows/manifest_test.yml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+on:
+#   push:
+#     branches: [master]
+#   pull_request:
+#     branches: [master]
+  issue_comment:
+    types: [created]
+
+jobs:
+  build_images_for_manifest_test:
```
Comment (eedorenko) [inline .github/workflows/manifest_test.yml:13]: Why is it called "build_images_..."? What images do we build?

Comment (yilun-msft) [reply .github/workflows/manifest_test.yml]: Changed.

File: .github/workflows/manifest_test.yml (hunk 1 above)
Comment (yilun-msft) [reply .github/workflows/manifest_test.yml]: Updated to use a shell script with env defined in yaml file.

**Review by eedorenko — APPROVED** [review]

**Pull request merged**